index.js

```diff
@@ -791,6 +791,17 @@ const _bindWindow = (window, newWindowCb) => {
       e.clipboardData.items.push(dataTransferItem);
     }
   });
+  window.document.on('copy', e => {
+    e.clipboardData = new window.DataTransfer();
+    if (contexts.length > 0) {
```

Comment:
We should probably fire either on all documents, or the "currently focused" one.